### ISSUE TITLE
[BUG_SP]

### ISSUE THREAD

**Issue by TheSnakeDoctor67** [Courseplay/Courseplay_FS22#1227]
**READ FIRST**
https://github.com/Courseplay/Courseplay_FS22/wiki

**Steps to reproduce**
Steps to reproduce the behavior.
Exit game and load saved game

**Actual behavior**
In game hud defaults to Left on all machines, does not save position when using multiple machines (center or right)

**Expected behavior**
In game hud should save position of multiple machines when saving, exiting and reloading a saved game

**Screenshots**

**Courseplay version**
Version used: 7.0.1.5

**Log**
Please attach a log with all debug channels active.
For help with the debug channels,
please check out the readme at the github front page.

**Map**
Map played on. 
All

**Vehicles/ implements involved**
Which vehicle or implement?
Mod and base game equipment

**Relevant script mods**
Autodrive
[log.txt](https://github.com/Courseplay/Courseplay_FS22/files/8287402/log.txt)
![Farming Simulator 22 Screenshot 2022 02 19 - 10 14 52 28](https://user-images.githubusercontent.com/97072517/158831851-112d7910-4a59-44ed-8e15-6d5724ce6798.png)

**Comment by TheSnakeDoctor67** [comment]
I am talking about the hud that comes up for course play when you click the
mouse or hit delete, example, if I have two combines harvesting a field
(course), 1st is left of the line, 2nd is right of the line, and I save the
game and exit when I return and restart the combines the RIGHT combine
defaults to the left side of the line instead of remembering that it is
supposed to be on the right side.

**Comment by Tensuko** [comment]
So it is not the hud, it is the lane offset.

**Comment by TheSnakeDoctor67** [comment]
Yes Courseplay is not remembering that the second and or third machine are set to center or right, it is defaulting to left, so if you save and exit the game when you load back in you have to go to the 2nd & 3rd machine and reset it for center or right on the course
![right](https://user-images.githubusercontent.com/97072517/159031919-cb8be2c5-6459-4abf-a7aa-79e9549a2665.png)
